compiler/luci-interpreter/pal/linux/PALConv2d.h

```diff
@@ -57,6 +58,22 @@ static inline void Conv(const tflite::ConvParams &params, const tflite::RuntimeS
                               im2col_data, gemmlowp_context.get());
 }
 
+static inline void ConvPerChannel(const tflite::ConvParams &params, const int32_t *mult,
+                                  const int32_t *shifts, const tflite::RuntimeShape &input_shape,
+                                  const int8 *input_data, const tflite::RuntimeShape &filter_shape,
+                                  const int8 *filter_data, const tflite::RuntimeShape &bias_shape,
+                                  const int32 *bias_data, const tflite::RuntimeShape &output_shape,
+                                  int8 *output_data, const tflite::RuntimeShape &im2col_shape,
+                                  int8 *im2col_data)
+{
+  (void)im2col_shape;
+  (void)im2col_data;
+  // TODO enable optimized version
+  tflite::reference_integer_ops::ConvPerChannel(params, mult, shifts, input_shape, input_data,
+                                                filter_shape, filter_data, bias_shape, bias_data,
+                                                output_shape, output_data);
+}
```

Comment:
Why can't we support the optimized version now? We can add 
`#include <tensorflow/lite/kernels/internal/optimized/integer_ops/conv.h>` and then change 
```suggestion
  tflite::optimized_integer_ops::ConvPerChannel(params, mult, shifts, input_shape, input_data,
                                                filter_shape, filter_data, bias_shape, bias_data,
                                                output_shape, output_data, im2col_shape, im2col_data);
}
```

Reply:
We can, but this requires linking with several new libraries from tensorflow, so I postponed it =)